### PR TITLE
Added some cameras on NSS Cyberiad

### PR DESCRIPTION
**Pull request by Fluffyzack21**
## What Does This PR Do
 Added some cameras around the station to help the AI not to have blind spots

## Why It's Good For The Game
 It will help the AI spot people being bad on NSS Cyberaid and tell over the radio who is doing a bad, and balances in medbay for antags and removes a lot of the privacy, due to adding camera coverage and removes blind spots from from the AI, so the AI can see what people are doing and such

## Images of changes
 
Screenshot 2023-11-30 222314
Screenshot 2023-11-30 222123
Screenshot 2023-11-30 214821
Screenshot 2024-01-29 180837

## Testing
 Went as the AI and checked if the cameras were working and they were

## Changelog
:cl:
add: Added cameras in, virology patient room 2, science break room and one in medbay on Box station
Tweak: Changed the wall near the OR2 to electro chromatic windows and a button to reflect it so you can change it on and off on box station
:cl:

### CODE REVIEW

**Comment by Fluffyzack21** [comment]
I may or may not just close this PR, I don't know

**Comment by 1080pCat** [comment]
I like this, its always a pain as ai when theres very strange blindspots.
I don't understand the downvotes and 😕 reactions, i assume its honestly just silicon hate.

**Comment by datlo** [comment]
I think the downvotes are because execution isnt supposed to have a camera.

**Comment by Fordoxia** [comment]
The custodial closet also lacks a camera. I regularly get asked to "fix the camera in the janitor closet"

**Comment by matttheficus** [comment]
> I like this, its always a pain as ai when theres very strange blindspots. I don't understand the downvotes and 😕 reactions, i assume its honestly just silicon hate.

> I think the downvotes are because execution isnt supposed to have a camera.

I’m with datlo here, it makes sense that Execution doesn’t have a camera - that and it makes sense in-world that Security doesn’t want video proof of the wacky ways they execute people sometimes. The other spots seem fine though, only one I’m iffy on is Execution.

**Comment by EmeraldCandy** [comment]
I think I also have issue with the operating theatre camera as well. In my opinion at least, its meant to be the private theatre for things that shouldnt be viewed by outsiders, hence why it has no windows. Its meant to try and help provide an area for medical antags to do some shady shit away from the prying eyes, including the AIs. I think leaving a camera out of it is ok, as I dont believe this one was left out my mistake, but by design. However, the other places are just fine I think. 

On a side note: yeah, the jani closet should probably also get a camera. Little reason to justify that one

**Comment by Fluffyzack21** [comment]
> I think I also have issue with the operating theatre camera as well. In my opinion at least, its meant to be the private theatre for things that shouldnt be viewed by outsiders, hence why it has no windows. Its meant to try and help provide an area for medical antags to do some shady shit away from the prying eyes, including the AIs. I think leaving a camera out of it is ok, as I dont believe this one was left out my mistake, but by design. However, the other places are just fine I think.
> 
> On a side note: yeah, the jani closet should probably also get a camera. Little reason to justify that one

I understand fully on the OR not having a camera, it allows for shady stuff to happen but for janitor closet. In Quotes by Hal 9000 "The janitor was intentionally left out because it's a "closet" and not really a room"

**Comment by meow20** [comment]
some places have intentional blindspots for antags, this pr is like putting up cameras in maints heh

**Comment by Denthamos** [comment]
Two things

First if making multiple changes please make sure to detail in words as well what you are changing like saying a list of the cameras to go along with the photos.

Secondly with the OR camera if anything it should match the other OR with chromatic windows facing the lobby with a button inside to black out the windows from AI/characters view.  The objects on the walls there currently would need to be moved elsewhere accordingly as well.

File: _maps/map_files/cyberiad/cyberiad.dmm [hunk 1]
```diff
@@ -85274,6 +85258,10 @@
 	pixel_y = -32
 	},
 /obj/item/trash/gum,
+/obj/machinery/camera{
+	dir = 8;
+	c_tag = virology
```
Comment (DGamerL) [inline _maps/map_files/cyberiad/cyberiad.dmm:85263]:
```suggestion
	c_tag = "virology"
```

Comment (Fluffyzack21) [reply _maps/map_files/cyberiad/cyberiad.dmm:85263]: fixed it

**Comment by matttheficus** [comment]
Objecting as Balance due to the cameras and windows in OR2.

As said above, some things are blind spots for a reason to allow for antags to have some safe areas to work in.

**Review by S34NW — CHANGES_REQUESTED** [review]
Please move the light from OR2's window onto the walls.

Your camera name tags are inconsistently capitalised, with inconsistent and sometimes poor names. Please see the mapping standards for camera name tags https://github.com/ParadiseSS13/Paradise/blob/master/.github/CONTRIBUTING.md#mapping-standards

**Comment by DGamerL** [comment]
> Why It's Good For The Game
It will help the AI spot people being bad on NSS Cyberaid and tell over the radio who is doing a bad

For the next time, this really is too short of an explanation, and not really a strong one. Please put a bit more effort into that section

File: _maps/map_files/cyberiad/cyberiad.dmm [hunk 2]
```diff
@@ -36151,6 +36151,9 @@
 /obj/machinery/computer/crew{
 	dir = 8
 	},
+/obj/machinery/camera{
+	c_tag = "Medbay"
```
Comment (S34NW) [inline _maps/map_files/cyberiad/cyberiad.dmm:36155]: More descriptive please, at least include where it is within medbay

**Review by S34NW — CHANGES_REQUESTED** [review]

**Comment by matttheficus** [comment]
So, gonna relay some of the Dev Team stuff here:
- Not a fan of the OR2 cam or windows - it’s the one place where Medbay antags can be evil.
- Not a fan atm either, Cameras in OR are agony and there shouldn't be one in Execution either; IMO.

There are currently two objections, one Balance and one Design.

**Comment by Fluffyzack21** [comment]
> So, gonna relay some of the Dev Team stuff here:
> 
> * Not a fan of the OR2 cam or windows - it’s the one place where Medbay antags can be evil.
> * Not a fan atm either, Cameras in OR are agony and there shouldn't be one in Execution either; IMO.
> 
> There are currently two objections, one Balance and one Design.

Okay I do not have cameras in the exection room, and no cameras in the OR

**Comment by SteelSlayer** [comment]
This PR has 3 objections, and will be closed. The consensus from the team is that adding windows to the OR room would destroy a lot of the privacy that medbay antags have.

**Pull request closed without merging**